code/modules/research/designs/mecha_designs.dm

```diff
@@ -48,7 +48,7 @@
 	name = "Exosuit Board (\"Gygax\" Central Control module)"
 	desc = "Allows for the construction of a \"Gygax\" Central Control module."
 	id = "gygax_main"
-	req_tech = list("programming" = 4, "combat" = 3, "engineering" = 4)
+	req_tech = list("programming" = 8, "combat" = 8, "engineering" = 8)
```

Comment:
Programming 8 is currently achievable by:
- `/obj/item/abductor/silencer` (agent abductor)
- `/obj/item/deactivated_swarmer` (swarmer spawner)
- `/obj/item/implant/weapons_auth` (currently unobtainable outside the syndicate research base)

Combat 8:
- `/obj/item/gun/energy/laser/instakill` (unobtainable)
- `/obj/item/gun/projectile/automatic/sniper_rifle` (only(?) comes from wizard summon guns spell)
- `/obj/item/gun/projectile/automatic/sniper_rifle/syndicate` (nuclear operative item)
- `/obj/item/organ/internal/cyberimp/arm/combat` (unobtainable)

Engineering 8 comes from:
- `/obj/item/abductor/gizmo` (scientist abductor)
- `/obj/item/assembly/signaler/anomaly/vortex` (anomaly event)

With this change, circuits are pretty much impossible to be ever achieved.

Reply:
This is more to make them unable to be created via RnD, but still easily obtainable via cargo.